lc0/src/mcts/search.cc

```diff
@@ -387,7 +387,7 @@ void Search::SendUciInfo() REQUIRES(nodes_mutex_) {
       cache_->GetSize() * 1000LL / std::max(cache_->GetCapacity(), 1);
   uci_info_.nps =
       uci_info_.time ? (total_playouts_ * 1000 / uci_info_.time) : 0;
-  uci_info_.score = -191 * log(2 / (best_move_node_->q * 0.99 + 1) - 1);
+  uci_info_.score = 290.680623072 * tan(1.548090806 * best_move_node_->q);
```

Comment:
Did you check it? I believe it will be sign inverted.
(`lc0` stores eval from the point of view of whose move it is, and position "after move" is opponent's move, so the sign should be flipped)

Reply:
Yep, I checked it. Working as expected on my bot and this shows the curves are similar and not inverted.

http://www.wolframalpha.com/input/?i=290.680623072+*+tan(1.548090806+*+x)+and+-191+*+log(2+%2F+(x+*+0.99+%2B+1)+-+1),+x+%3D+%5B-1,1%5D